new-saudi-riyal-for-woocommerce.php

```diff
@@ -79,3 +80,74 @@ function nsrwc_replace_sar_currency_symbol( $currency_symbol, $currency ) {
 }
 
 add_filter( 'woocommerce_currency_symbol', 'nsrwc_replace_sar_currency_symbol', 10, 2 );
+
+/**
+ * Force left position for SAR currency
+ */
+function nsrwc_set_currency_position_left( $position ) {
+    if ( 'SAR' === get_woocommerce_currency() ) {
+        return 'left';
+    }
+    return $position;
+}
+add_filter( 'woocommerce_currency_position', 'nsrwc_set_currency_position_left' );
```

Comment:
I understand that the symbol should be in the left but i think keeping it under admin control it better than overriding this setting.
```suggestion
```